### ISSUE TITLE
Call to undefined method LabelDelivery::preventsAccessingMissingAttributes()

### ISSUE THREAD

**Issue by codecreativedm** [rawilk/laravel-ups#36]
### Laravel UPS Version

v2.1

### Laravel Version

v10.42.0

### Bug description

Just recently, we have been getting a "preventsAccessingMissingAttributes" when trying to create a shipment. Our code is basically a lift of the code in the documentation (as that fits our needs). 

I can't find the route cause of this error. Based on my understanding, preventsAccessingMissingAttributes() is related to Laravel models and error management for unselected columns in a query. I'm not sure how this applies in this case as the creation of a Shipment doesn't directly interact with a model. My only thought that the Entity class is somehow involved in this error. 

In my testing, it's not just the LabelDelivery method that is throwing the error; we experienced the same issue when generating payment information with the PaymentInformation method.  

The Laravel 10 may also be the related. We are using laravel-ups in two separate apps, one running Laravel 9.52.16 and another running 10.42.0; at this time we are only experiencing the error in Laravel 10.42 

### Steps to reproduce

This error is triggered when creating a new Shipment, with a label, and generating a digest (getDigest).

### Relevant log output

```shell
Call to undefined method Rawilk\\Ups\\Entity\\Shipment\\ShipmentServiceOptions\\LabelDelivery::preventsAccessingMissingAttributes()
```

**Comment by codecreativedm** [comment]
I discovered a workaround for this. I suspected the Entity model was partly the issue as it has similar properties as a model but not officially a model. And, since it's not a model, it wouldn't have access to the preventsAccessingMissingAttributes method introduced in Laravel 9.x. 

I simply added the following to the Entity class and now able to generate a shipping digest. 

`public static function preventsAccessingMissingAttributes() { return false; }`

While this is not a perfect solution, it will hold us over until something official is provided.
